### PR TITLE
dp/merchant relationships

### PR DESCRIPTION
**Pull request by Dpalazzari**
@tmikeschu 
@sallymacnicholas 

This PR:

- Adds items relationship for api/v1/merchants API
- Adds invoices relationship for api/v1/merchants API
- All tests pass

### CODE REVIEW

**Comment by Dpalazzari** [comment]
Yo @tmikeschu so this was pretty simple, already passing the test and it works on Rails s. What do you think of the test case?

File: app/controllers/api/v1/merchants/merchant_invoices_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Api::V1::Merchants::MerchantInvoicesController < ApplicationController
+
+	def index
+		render json: Merchant.find(params[:id]).invoices
```
Comment (tmikeschu) [inline app/controllers/api/v1/merchants/merchant_invoices_controller.rb:4]: @Dpalazzari what's your thought on two-lining this to
```
merchant = Merchant.find(params[:id])
render json: merchant.invoices
```
?

More verbose, but maybe more readable?

File: config/routes.rb [hunk 2]
```diff
@@ -11,6 +11,8 @@
         get '/find', to: 'search#show'
         get '/find_all', to: 'search#index'
         get '/random', to: 'random#show'
+        get ':id/items', to: 'merchant_items#index'
```
Comment (tmikeschu) [inline config/routes.rb:14]: Love the SRP on the controllers! 👍

**Review by tmikeschu — COMMENTED** [review]
Looks great! let's talk in person about the comment I left about two-lining the show action. I'm ready to merge!

File: app/controllers/api/v1/merchants/merchant_invoices_controller.rb (hunk 1 above)
Comment (Dpalazzari) [reply app/controllers/api/v1/merchants/merchant_invoices_controller.rb:4]: I can do that. I'm making another PR right now for invoices. Ill make this change.
@tmikeschu

**Pull request merged**